python/config.py.in

```diff
@@ -0,0 +1,8 @@
+import os
+
+MLIR_C_RUNNER_UTILS = os.path.normpath(
```

Comment:
It would be better to just register a folder path here, maybe `MLIR_SHARED_LIB_PATH`, and then in the `graph_compiler.py file`, you can search the lib name in an OS independent way.

Reply:
We can set mlir_runner_utils_dir to `@MLIR_RUNNER_UTILS_DIR@` for our `lit.site.cfg.py.in`. Can we use that cmake variable too?

Reply:
I tried and we can not get the `@MLIR_RUNNER_UTILS_DIR@`  from here, not in the same scope.